copy_encrypted_ami.sh

```diff
@@ -113,7 +113,7 @@ AMI_DETAILS=$(aws ec2 describe-images --profile ${SRC_PROFILE} --region ${SRC_RE
 
 
 # Retrieve the snapshots and key ID's
-SNAPSHOT_IDS=$(echo ${AMI_DETAILS} | jq -r '.BlockDeviceMappings[].Ebs | .SnapshotId' || die "Unable to get the encrypted snapshot ids from AMI. Aborting.")
+SNAPSHOT_IDS=$(echo ${AMI_DETAILS} | jq -r '.BlockDeviceMappings[] | select(has("VirtualName") | not) | .Ebs.SnapshotId' || die "Unable to get the encrypted snapshot ids from AMI. Aborting.")
```

Comment:
Why not "select(has("Ebs"))" instead of "select(has("VirtualName") | not)" ? I understand that currently VirtualName is only for Ephemeral, but in case AWS decide to use it to EBS for some reason, than the script will stop working. Do you see any downside using "select(has("Ebs"))"  ?

Reply:
Hi, yes, I'm sure that'll be better that my approach, but I haven't had a chance to give it a try as yet. I should get a chance in the next week or so.

Reply:
Thanks @man8 ! I have tested my self with "select(has("Ebs"))" and it works. If you don't have time next week please let me know and I will submit another PR for this.

Reply:
Thanks @man8 ! I have tested my self with "select(has("Ebs"))" and it works. If you don't have time next week please let me know and I will submit another PR for this.